.travis.yml

```diff
@@ -26,6 +26,11 @@ jobs:
     dist: xenial
     env:
     - PACKAGE=misc/magic
+  - stage: "No dependencies"
+    os: linux
+    dist: xenial
+    env:
+    - PACKAGE=misc/python-constraint
```

Comment:
this should, probably, go to litex-conda-misc, but since it's a direct dependency of vtr-optimized, I don't think it's a big problem to have it here